### ISSUE TITLE
Investigate what Exodus transactions that effect token amounts

### ISSUE THREAD

**Issue by ultimaweapon** [zcoinofficial/ztm#99]
Create a new issue for the investigation result. It must contains:

- Different between each tally type and which one we are interest.
- Transaction type and version that effect balance of the tally type that we interested.
- What condition it will changes the balance.
- Is there any possible cases that the balance will be changed if transaction is failed (transaction not stored in Exodus database, thus cannot get that transaction via RPC).

- [x] Investigate [3]

**Comment by ultimaweapon** [comment]
Blocked by #79.

**Comment by ultimaweapon** [comment]
See the detailed in #105.
